### ISSUE TITLE
general site glossary

### ISSUE THREAD

**Issue by MortenHofft** [gbif/portal16#240]
We use a lot of terms that are unfamiliar to people outside the network. 

A discussion about how much we believe is necessary to explain and how and where to do so is needed. E.g. do we believe that Types (holotype etc) needs further explaining or do we assume that our audience know them? 

I for one believe that an explanation would be useful - even for biologist (can you remember [all types](http://api.gbif.org/v1/enumeration/basic/TypeStatus) for example? What is a hapantotype?)

@dschigel, @kbraak and @kcopas I got the feeling you all had given this some thought

**Comment by dschigel** [comment]
Yes, let's start collecting the terms in need for definition. Perhaps a convenient way to populate the list is to start a table alphabetically sorted by term/idiom, then have fields:category (e.g. taxonomy, IPT, informatics, ecology, DwC)

**Comment by MortenHofft** [comment]
from user http://dev.gbif.org/issues/browse/PF-2645?jql=project%20%3D%20PF%20AND%20status%20%3D%20Open
